### PR TITLE
feat(search) Adding search results from generic

### PR DESCRIPTION
**Pull request by stevenpkg**
* Added search skills results to chat area
* Added full search docs to open in a new browser tab

### CODE REVIEW

**Comment by stevenpkg** [comment]
Adding @Oliver-I

**Comment by stevenpkg** [comment]
@germanattanasio not sure why this is coursing a problem?

```
Module build failed: ModuleBuildError: Module build failed: 
@import '../components/SearchResults/SearchResultsList';
```

File: src/components/SearchResults/SearchresultsList.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.search-item {
```
Comment (germanattanasio) [inline src/components/SearchResults/SearchresultsList.scss]: I don't see why this is not working 🤷‍♂

File: src/components/App.js [hunk 2]
```diff
@@ -56,6 +55,18 @@ const parseNonGenericResponsesFromBotOutput = (genericObj) => {
         type: 'image',
         content: response.source,
       });
+    } else if (response.response_type === 'search') {
+      responses.push({
+        type: 'bot',
+        content: response.header,
+      });
+      // response.results.forEach((res) => {
```
Comment (germanattanasio) [inline src/components/App.js]: remove commented code

File: index.js [hunk 3]
```diff
@@ -140,4 +169,23 @@ app.get('/api/session', (req, res) => {
   });
 });
 
+app.get('/search', (req, res) => {
+  const id = req.query.id;
+
+  let header = searchCache.get(`header_${id}`);
+  let body = searchCache.get(`body_${id}`);
+
+  if (header === undefined) {
```
Comment (germanattanasio) [inline index.js]: you can just do `if (header)`

File: index.js [hunk 4]
```diff
@@ -94,12 +126,9 @@ app.post('/api/message', (req, res) => {
   // send payload to Conversation and return result
   assistant.message(payload, (err, data) => {
     if (err) {
-      console.log(err);
```
Comment (germanattanasio) [inline index.js]: you probably want to keep this in case we need to debug the app.
if you are getting eslint issues just add `//eslint-disable-next-line`

**Comment by stevenpkg** [comment]
@germanattanasio thanks for your good feedback, have made the changes you requested. Also fixed the Travis error. It was a case issue Search**r**esultsList.scss should be Search**R**esultsList.scss, did not course a problem on Mac when running tests

**Pull request merged**